### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by jwillmer** [martinriedel/syncthing.net#1]
I see the repo is pretty new so I think I had some luck. I was about to evaluate if I will build my own SDK for syncthing. Thank you for your work!

Two questions about your code:

- How much feature complete is it? Did you only got started or ist the deployment to GitHub the last step you did?
- I saw the note "based on octokit". What does this mean? I have no idea / never used octokit.

**Comment by martinriedel** [comment]
I was in the same situation like you just a few days ago. At the moment I only started with very basic features. But my plan ist to be feature complete in a few weeks because I something like an SDK for my work.

- I just started last weekend and I will try to continue development mainly in my free time. 
- Octokit is the official .NET SDK for Github and I like the architectural structure they used. I have taken the basic functions and now I try to extend everything to work with Syncthing
